extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -0,0 +1,260 @@
+        },
+        "https://ns.adobe.com/experience/campaign/message/seedMember": {
+          "title": "Seed Member",
+          "type": "boolean",
+          "default": false,
+          "description":
+            "The recipient of this message is a member of a seed list for this campaign activity."
+        },
+        "https://ns.adobe.com/experience/campaign/message/quarantine": {
+          "title": "Quarantine",
+          "type": "boolean",
+          "default": false,
+          "description":
+            "Address moved to quarantine. The quarantine is set to true when the message failed to deliver in a way that indicates that no future messages will be deliverable either to this address."
+        },
+        "https://ns.adobe.com/experience/campaign/message/proofMember": {
+          "title": "Proof Member",
+          "type": "boolean",
+          "description":
+            "The recipient of this message is a member of the proof group for this campaign activity.",
+          "default": false
+        },
+        "https://ns.adobe.com/experience/campaign/message/controlGroupMember": {
+          "title": "Control Group Member",
+          "type": "boolean",
+          "description":
+            "The recipient of this message is a member of a control group for this campaign activity. Messages to control group members are not delivered.",
+          "default": false
+        },
+        "https://ns.adobe.com/experience/campaign/message/testMember": {
+          "title": "A/B Test Member",
+          "type": "boolean",
+          "description":
+            "The recipient of this message is a member of a test group for this campaign activity.",
+          "default": false
+        },
+        "https://ns.adobe.com/experience/campaign/message/size": {
+          "title": "Size (Bytes)",
+          "type": "integer",
+          "description":
+            "Size in bytes of the message.\n* For email, the `size` reflects the all MIME envelop of the message, including the encoded attachments if any.\n* For SMS, the `size` reflects the number of bytes necessary to deliver the message.\nIf that number is over 160 the message will be delivered over more than one SMS.",
+          "minimum": 0,
+          "maximum": 2147483647
+        },
+        "https://ns.adobe.com/experience/campaign/message/reason": {
+          "title": "Reason",
+          "type": "string",
+          "meta:enum": {
+            "Undefined": "Not defined",
+            "UnknownUser": "User unknown",
+            "InvalidDomain": "Invalid domain",
+            "Unreachable": "Unreachable",
+            "Disabled": "Account disabled",
+            "MailboxFull": "Mailbox full",
+            "NotConnected": "Not connected",
+            "Refused": "Refused",
+            "ErrorIgnored": "Error ignored",
```

Comment:
What does that mean?

Reply:
This means that there was an error while sending to the recipient but since the user is whitelisted, we will ignore the error and not mark them as blacklisted. I will update this in the description.
For more: https://docs.campaign.adobe.com/doc/AC/en/DLV_Tracking_deliveries_Quarantine_management.html